### PR TITLE
Creates an indexed table view

### PR DESCRIPTION
**Pull request by lauyoo46**
image

### CODE REVIEW

File: IndexedTableDemo/AnimalTableViewController.swift [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        return index
+    }
+    
+    override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 50
+    }
+    
+    override func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
+        if let headerView = view as? UITableViewHeaderFooterView {
+            headerView.backgroundView?.backgroundColor = UIColor(red: 236.0/255.0,
```
Comment (czegreanu) [inline IndexedTableDemo/AnimalTableViewController.swift]: please extract these colors in an enum

File: IndexedTableDemo/AnimalTableViewController.swift [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        
+        if let animalValues = animalsDict[animalKey] {
+            cell.textLabel?.text = animalValues[indexPath.row]
+            let imageFileName = animalValues[indexPath.row].lowercased().replacingOccurrences(of: " ", with: "_")
+            cell.imageView?.image = UIImage(named: imageFileName)
+        }
+        
+        return cell
+    }
+
```
Comment (czegreanu) [inline IndexedTableDemo/AnimalTableViewController.swift]: delete line

File: IndexedTableDemo/AppDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+//  Copyright © 2016 AppCoda. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
```
Comment (czegreanu) [inline IndexedTableDemo/AppDelegate.swift]: delete line

File: IndexedTableDemo/AppDelegate.swift [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    func applicationWillEnterForeground(_ application: UIApplication) {
+        // Called as part of the transition from the background to the active state; here you can undo many of the changes made on entering the background.
+    }
+
+    func applicationDidBecomeActive(_ application: UIApplication) {
+        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
+    }
+
+    func applicationWillTerminate(_ application: UIApplication) {
```
Comment (czegreanu) [inline IndexedTableDemo/AppDelegate.swift]: delete methods that are not needed

**Review by czegreanu — CHANGES_REQUESTED** [review]

**Review by czegreanu — APPROVED** [review]

**Review by a-nistor — APPROVED** [review]

**Pull request merged**